docs/onboarding/onboarding-feature-ios.md

```diff
@@ -0,0 +1,249 @@
+| set-default-browser        | Will send the user to settings to set Firefox as their default browser              |
+| open-default-browser-popup | Will open up a popup with instructions for setting Firefox as their default browser |
+| read-privacy-policy        | Will open a webview where the user can read the privacy policy                      |
+
+## Dismissable
+This is a property for the whole onboarding, and controls whether there is an `x` at the top of the screen or not. This is a simple boolean value. ex:
+
+```json
+{
+    "dismissable": true
+}
+```
+
+# Available resources
+
+## TextID
+All existing app strings are available to Experimenter. [See the full list here](https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/Strings.swift). Note, that only strings with a `tableName` and a `key` can be used.
+Free text may also be used instead of a string resource.
+
+⚠️ Localization of **Free Text** is not currently supported.
+
+## Image resources
+
+### Base Resources
+| Resource ID   | Description                                   |
+|:--------------|:----------------------------------------------|
+| welcome-globe | An image of a person hugging the firefox logo |
+| sync-devices  | An image of a variety of devices              |
+| notifications | An image of notifications                     |
+
+### Campaign - Challenge the Default
+
+| Resource ID      | Description                                                                                                                                                                  |
+|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| welcome-ctd      | The welcome image for CTD campaign. |
+| sync-devices-ctd | The sync image for CTD campaign. |
+| notification-ctd | The notifications image for CTD campaign. |
+
+# Appendix
+
+## Default cards overview
+
+### Default Welcome Card
+| Attribute     | Value                                                 |
+|---------------|-------------------------------------------------------|
+| type          | fresh-install                                         |
+| title         | Onboarding/Onboarding.Welcome.Title.v114              |
+| body          | Onboarding/Onboarding.Welcome.Description.v114        |
+| link          | title: Onboarding/Onboarding.Welcome.Link.Action.v114 |
+|               | url: "https://www.mozilla.org/privacy/firefox/"       |
+| image         | welcome-globe                                         |
+| ordering      | 10                                                    |
+| buttons       |                                                       |
+| - primary     | title: Onboarding/Onboarding.Welcome.Action.v114      |
+|               | action: next-card                                     |
```

Comment:
Should `url:` and `action` be in the first column here? 
<img width="651" alt="image" src="https://github.com/mozilla/experimenter-docs/assets/43795363/61fdd3ad-4de3-4540-8108-1d838659036d">

Reply:
Same question for `Default Sync card` and `Default Notification card`

Reply:
Dang it! I mixed up two things. I'll fix it!